### PR TITLE
Check contenteditable attribute

### PR DESCRIPTION
**Pull request by jamesfzhang**
Addresses https://github.com/ayrton/react-key-handler/issues/42.

### CODE REVIEW

**Comment by ayrton** [comment]
Hey James thanks for this patch, I'll have a look at this soon.

File: lib/utils.js [hunk 1]
```diff
@@ -86,8 +86,14 @@ export function isInput(element: HTMLElement): boolean {
   }
 
   const {tagName} = element;
+  let editable;
+  if (typeof element.getAttribute === 'function') {
+    editable = !!element.getAttribute('contenteditable');
+  } else {
+    editable = false;
+  }
```
Comment (ayrton) [inline lib/utils.js]: I think it makes sense to extract this to a different function:

```js
export function isInput(element: HTMLElement): boolean {
   if (!element) {
     return false;
    }

  const {tagName} = element;
  const editable = isEditable(input);

  return tagName === 'INPUT' || tagName === 'TEXTAREA' || editable;
}

function isContentEditable(element: HTMLElement): boolean {
  if (typeof element.getAttribute !== 'function') {
    return false;
  }

  return !!element.getAttribute('contenteditable');
}
```

File: lib/utils.js [hunk 2]
```diff
@@ -86,8 +86,14 @@ export function isInput(element: HTMLElement): boolean {
   }
```
Comment (ayrton) [inline lib/utils.js]: Could you also update the jsdoc to something in the lines of:

```js
 /**
  * Check if `given` element is an input / textarea form element or acts as one.
  */
```

**Comment by ayrton** [comment]
@jamesfzhang looking awesome, I think with the feedback I left in the PR this is good to merge in - could you also rebase your branch please. Thanks for your contribution 💯

**Comment by jamesfzhang** [comment]
@ayrton Thanks for the feedback, PR should good to go!

**Pull request merged**